search.go

```diff
@@ -184,19 +184,19 @@ func (s *Search) FullSearch(query string) ([]*Node, int, time.Duration) {
 	bSearch := bleve.NewSearchRequest(disjunctionQuery)
 	searchResults, err := s.index.Search(bSearch)
 	if err != nil {
-		log.Fatalf("Query: '%s' failed...", query)
+		return nil, 0, time.Duration(0), fmt.Errorf("Query '%s' failed: %s", query, err)
```

Comment:
We probably don't want a time of 0 thrown out here, but instead the amount of time the failure took?